apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/style.scss

```diff
@@ -30,17 +52,74 @@ $wpcom-modal-padding-h: 50px;
 		margin-top: 0;
 	}
 
-	.components-modal__content {
-		padding: $wpcom-modal-padding-v $wpcom-modal-padding-h;
+	.components-guide__footer {
+		position: absolute;
+		width: 100%;
+		height: $wpcom-modal-footer-height;
+		bottom: $wpcom-modal-footer-height * -1;
+		left: 0;
+		padding: $wpcom-modal-footer-padding-v 0;
+		margin: 0;
+		display: flex;
+		justify-content: center;
+		background: white;
+		border-top: 1px solid #dcdcde;
```

Comment:
Any Gutenberg sass variable we could use here? From `@import '~@wordpress/base-styles/colors';`

Reply:
sounds good 👍